src/components/basic/containers/Scrollable.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import classnames from "classnames";
+
+const Scrollable = props => {
+  const { children, className, ...rest } = props;
```

Comment:
What does the `...rest` variable do?

Reply:
@theilman7 here is a pretty good explanation https://stackoverflow.com/a/43484565/1143670. 

In this case we are trying to make this container pretty flexible, so this allows us to add attributes to the container. 

The more you work with React, you will see this pattern used by others. It's not something I personally have a definitive article on (like this is the way you do it) but with practice has been something I personally have been doing.

Reply:
Thanks for providing that information Marty!